src/osgPlugins/dxf/dxfEntity.cpp

```diff
@@ -910,10 +911,19 @@ dxfText::drawScene(scene* sc)
     sc->ocs(m);
 
     ref_ptr<osgText::Text> _text = new osgText::Text;
-    _text->setText(_string);
+    if (_useWideChar && !_string.empty())
```

Comment:
Are DXF text strings always encding in UTF8? if so we can just use line

  _text->setText(osgText::String(_string, osgText::String::ENCODING_UTF8));

In place of the original _text->setText(_string) line.